pages/mesosphere/dcos/2.1/deploying-services/pods/technical-overview/index.md

```diff
@@ -71,6 +71,25 @@ PWD=/mnt/mesos/sandbox
 EP_HOST_HTTPENDPOINT=21528
 ```
 
+# Resource Requests and Limits
+
+Each container in a pod will require some resources to execute. The `resources` field contains an object with fields specifying the quantities of various resources required by the container. These quantities are known as the resource "requests", and the container is guaranteed to have access to at least these amounts of each resource. In addition, DC/OS has the concept of CPU and memory "limits" which allow you to specify the maximum amount of CPU or memory that the app may consume. The resource limit must be equal to or greater than the request; in addition, the limit may be given the value of "unlimited", meaning the app's use of that resource is not constrained at all. The app below has an infinite CPU limit and a memory limit of 4096 MB:
```

Comment:
Nope that isn't true, that depends on the value of the `shareCgroups` field. I decided not to touch on the `shareCgroups` stuff at all, since we're doing `shareCgroups=false` by default now and we don't expose it via the UI.

Reply:
Hmm, I think we may still want to have a section on shareCgroups so that users can understand what changes are happening to their containers when they remove the flag "shareCgroups".